LibSWBF2/Chunks/HeaderNames.cpp

```diff
@@ -1,13 +1,25 @@
 #include "pch.h"
 #include "HeaderNames.h"
 #include "Types/LibString.h"
+#include "Hashing.h"
 #include <set>
 #include <array>
+#include <map>
 #include <string>
 
 namespace LibSWBF2
 {
-	const std::set<ChunkHeader> KNOWN_HEADERS =
+	const std::map<ChunkHeader, std::string> KNOWN_SOUND_HEADERS = 
+	{
+		{"StreamList"_fnvh, 	"StreamList"},
+		{"Stream"_fnvh, 		"Stream"},
+		{"Info"_fnvh, 			"Info"},
+		{"SoundBankList"_fnvh, 	"SoundBankList"},
+		{"Data"_fnvh, 			"Data"},
+		{"SampleBank"_fnvh, 	"SampleBank"},
+	};
+
+	const std::set<ChunkHeader> KNOWN_GENERIC_HEADERS =
```

Comment:
Kept a separate mapping for sound headers to their original strings could be retrieved easily